packages/core/src/components/search/Filter/FilterSlider.tsx

```diff
@@ -227,18 +229,26 @@ function FilterSlider({
             const index = shouldDisplayDeliveryButton ? idx + 1 : idx
             const { __typename: type, label } = facet
             const isExpanded = expanded.has(index)
-            const isDeliveryFacet = facet.key === 'shipping'
+            const isDeliveryMethodFacet = facet.key === 'shipping'
+            const isDeliveryOptionFacet = facet.key === 'delivery-options'
+
+            let sectionLabel = label
+            if (isDeliveryMethodFacet) {
+              sectionLabel = deliveryMethodsLabel
+            } else if (isDeliveryOptionFacet) {
+              sectionLabel = deliveryOptionsLabel
+            }
 
             return (
               <UIFilterFacets
-                key={`${testId}-${label}-${index}`}
+                key={`${testId}-${sectionLabel}-${index}`}
                 testId={`mobile-${testId}`}
                 index={index}
                 type={type}
-                label={isDeliveryFacet ? deliveryLabel : label}
+                label={sectionLabel}
                 description={
-                  isDeliveryFacet
-                    ? deliverySettingsData?.description
+                  isDeliveryMethodFacet
+                    ? deliverySettings?.description
```

Comment:
It should be `deliverySettingsData` because we should use the return from the `getRegionalizationSettings` function:
```
const regionalizationData = getRegionalizationSettings({
    deliverySettings,
  })
const { deliverySettings: deliverySettingsData } = regionalizationData
```
```suggestion
                    ? deliverySettingsData?.description
```

Reply:
Done in here https://github.com/vtex/faststore/pull/2930/commits/1247be475ab462d1f5eb42867f9a4c5e07fb7944